src/module.c

```diff
@@ -33,8 +33,8 @@
  * The comments in this file are used to generate the API documentation on the
```

Comment:
This file is far too large to realistically review right now. I'm going to assume everything in this file was just a find/replacement outside of the few strategic compatibility hooks.

Reply:
@madolson I'm going through it. I've made it half-ways so far. I'm not being very strict about correct English. It's code comments, not end-user docs.

Reply:
load and unload hooks are updated. Others are indeed just find/replace. Maybe I should run clang-format on module.c now? :-)

Reply:
Eh, we're completely destroying git history, we can do it later and destroy it again.